src/components/ProfileForm.jsx

```diff
@@ -1,75 +1,224 @@
 import { FiEdit2 } from 'react-icons/fi';
+import { useSelector } from 'react-redux';
+import {
+  selectCurrentUser,
+  selectCurrentToken,
+  updateUser,
+} from '../features/auth/authSlice';
```

Comment:
You can explore absolute imports, it makes the code look cleaner
[https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8](url)

Reply:
Sure , Thanks for the suggestions . Done making the suggested changes